### DISCUSSION TITLE
Import PDF on top layer

### DISCUSSION THREAD

**Discussion by LeviticusMaximus** [pdfme/pdfme#157] [Q&A]
Hi,

I would like to insert content images onto a page, and them import a pdf with a cutcontour (spot colour vector lines) on the top of the imagery [Ideally overprinting it]. The imported PDF is the same size exactly as the base PDF page.

Is this possible?

Thanks in advance

**Comment by hand-dot** [comment]
Hi @LeviticusMaximus !

Sorry, at this moment, we can't. 
as a workaround, Why not just use another pdf library to layer on top of basePDF?
